packages/manager/apps/web/client/app/hosting/website-coach/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,12 @@
+{
+  "hosting_website_coach_title": "Website Coach",
+  "hosting_website_coach_button": "Activer Website Coach",
+  "hosting_website_coach_1dn_text1": "Recevez régulièrement des conseils basés sur une analyse à 360° de votre site",
+  "hosting_website_coach_1dn_text2": "sur l'adresse email",
+  "hosting_website_coach_1dn_text3": "pour en améliorer votre audience, la sécurité et plus généralement la satisfaction de vos visiteurs.",
+  "hosting_website_coach_mdn_text1": "Recevez régulièrement des conseils basés sur une analyse à 360° de votre site hébergé chez OVH sur l'adresse email",
+  "hosting_website_coach_mdn_text2": "pour en améliorer votre audience, la sécurité et plus généralement la satisfaction de vos visiteurs.",
```

Comment:
Looks like a duplicate of https://github.com/ovh-ux/manager/pull/1390/files#diff-fb06d2d12ee3f8c81fcb9f3e61c3387fR6

Reply:
Yes I know, but it's voluntary in case the text should change in the future for this case

Reply:
I've updated to a single translations with values